### ISSUE TITLE
Is form feed a non-permitted line ending according to specification? 

### ISSUE THREAD

**Issue by kaddkaka** [nvim-neorg/norg-specs#27]
I'm trying to read the 1.0 specification for the first time, but it's a bit hard to digest. 

According to the excerpt below, can I use form feed as a line ending or not?

```
The following chars are considered line endings:
    - A line feed `U+000A`
    - A form feed `U+000C`
    - A carriage return `U+000D`

    The following line ending combinations are permitted:
    - A single line feed
    - A single carriage return
    - A carriage return immediately followed by a line feed
```

**Comment by mrossinek** [comment]
No, the form feed is the page break character. That means, when a document gets displayed on pages, it will forces whatever comes after it, to start on the next page.
However, this does not mean that the line (or paragraph for that matter) is considered ended/broken, even though it has to be displayed as such (due to the paged output).

See also this document: https://www.unicode.org/standard/reports/tr13/tr13-5.html, especially recommendation 4.5 at the bottom.

I am leaving this issue open because we could be more explicit about this in the spec.
